testing/cli-e2etest/testscenarios/demo/delete_demo_test.go

```diff
@@ -28,7 +28,7 @@ func TestDeleteDemo(t *testing.T) {
 	// Then it should return an error and say that this resource does not exist
 	result := tracetestcli.Exec(t, "delete demo --id some-demo", tracetestcli.WithCLIConfig(cliConfig))
 	helpers.RequireExitCodeEqual(t, result, 1)
-	require.Contains(result.StdErr, "Resource demos with ID some-demo not found") // TODO: update this message to singular
+	require.Contains(result.StdErr, "Resource demo with ID some-demo not found") // TODO: update this message to singular
```

Comment:
❤️ , now we can remove this TODO 😄 
```suggestion
	require.Contains(result.StdErr, "Resource demo with ID some-demo not found") 
```